src/navigators/PopupWindow.ts

```diff
@@ -31,6 +31,13 @@ export class PopupWindow extends AbstractChildWindow {
         super();
         const centeredPopup = PopupUtils.center({ ...DefaultPopupWindowFeatures, ...popupWindowFeatures });
         this._window = window.open(undefined, popupWindowTarget, PopupUtils.serialize(centeredPopup));
+        if (popupWindowFeatures?.closePopupWindowAfter && popupWindowFeatures?.closePopupWindowAfter > 0) {
```

Comment:
I guess:
        `if (popupWindowFeatures?.closePopupWindowAfter && popupWindowFeatures?.closePopupWindowAfter > 0) {`
->
        `if (popupWindowFeatures.closePopupWindowAfter > 0) {`

Reply:
I have error possibily 'undefined' so i have to check if the parameter is defined. It is because of screen below
![obraz](https://github.com/authts/oidc-client-ts/assets/58402087/abd2094b-4c1f-448d-8a5e-68ef891d3738)

I can change it to not allow undefined values and I think I will do that